Gruntfile.js

```diff
@@ -116,7 +116,7 @@ module.exports = function( grunt ) {
 				},
 				frameworks: [ "qunit" ],
 				files: [
-					"https://code.jquery.com/jquery-git.min.js",
+					"https://code.jquery.com/jquery-3.4.1.min.js",
```

Comment:
A replacement preserving the original spirit would be:
https://code.jquery.com/jquery-3.x-git.min.js
We already use it in TestSwarm, see:
http://swarm.jquery.org/job/9162

Reply:
Yeah, I had been playing with the versions to make sure it wasn't something else and forgot to push the 3.x-git version. Updated now.